### DISCUSSION TITLE
Is there a way to allow specific commands when extension is not activated ?

### DISCUSSION THREAD

**Discussion by adhamfarrag** [microsoft/vscode-discussions#965] [Extension Development QnA]
It's stupid question, but wanted to make sure there's not way to do so except with refactoring activation and hiding components (commands, sidebar, etc).

**Comment by adhamfarrag** [comment]
Context: I'm trying to introduce a Nuxt project creation command that works without having to leave the extension activated. https://github.com/nuxtrdev/nuxtr-vscode/pull/95. While all commands are hidden when project / workspace isn't a nuxt project except for this one.

When I try to run it, it gives me an error that the command doesn't exist.

**Comment by olguzzar** [comment]
Hi @adhamfarrag, if I'm understanding your question correctly, this would not be possible with the extension not being activated. I see that the issue you linked is also closed, so I'm going to close this discussion - but if there's anything new to add, please feel free to re-open!
